neuro_flow/batch_runner.py

```diff
@@ -106,47 +186,77 @@ async def bake(self, batch_name: str) -> None:
         # check fast for the graph cycle error
         toposorter.prepare()
 
-        click.echo("Config is correct")
+        configs = [config_file, *await self._collect_additional_configs(flow)]
+        configs_data = [
+            (
+                RemotePath(path.relative_to(self._config_dir.workspace)),
```

Comment:
ConfigDir.workspace is a `LocalPath` instance.
Why do you create `RemotePath` here?